ShopGo/LoginScreen/TextField+extension.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  TextField+extension.swift
+//  ShopGo
+//
+//  Created by Waqas Shabir on 7/25/23.
+//
+
+import UIKit
+extension UITextField {
+    
+    func leftImage(imageViewNamed: String) {
```

Comment:
This can be used in the future so give it a generic name rather than leftImage

Reply:
changed it to iconInsideTextField